Sources/Clappr_iOS/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -450,16 +450,36 @@ open class AVFoundationPlayback: Playback {
         seekOnReadyIfNeeded()
 
         if let subtitles = self.subtitles {
-            trigger(.subtitleAvailable, userInfo: ["subtitles": subtitles])
+            let hasDefault = selectDefaultSubtitleIfNeeded()
+            trigger(.subtitleAvailable, userInfo: ["subtitles": subtitles, "hasDefaultFromOption": hasDefault])
```

Comment:
`hasDefaultFromOption` is true if the user passed any audio/subtitle preference and it was selected, correct?

Maybe we can change this to `defaultSubtitleSelected`.